recipes/entwine/meta.yaml

```diff
@@ -0,0 +1,57 @@
+
+{% set name = "entwine" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "ef1fc45ac0553fbbebf4d1f0ff3fbe38cdb0e6f7875adac1c8fb57ef657bd48e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/connormanning/entwine/archive/{{version}}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - pdal 1.8.*
+    - gdal 2.3.*
```

Comment:
This should be left unpinned so conda-smithy can do the pinning updates here automatically. (Which is 2.3 at the moment BTW.)

Same for all other pinnings except for those that are not specified in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml or there is a strong reason to go against the central pinning.

Reply:
> This should be left unpinned 
For all packages, or just PDAL and GDAL?

Reply:
All packages that are not in the central pinning file I mentioned above or that needs special pinning that is different from the central pinning. I don't think pdal is listed there so you may want to keep that one or, better yet, send a PR to the central pinning file.

Reply:
Since we're the PDAL developers maintaining the Conda PDAL recipe, if we break something, we'll upstream a fix in PDAL. We do work hard to preserve compatibility.

Reply:
Looks like the recipe is not OK with our latest pinning but the error code is not helpful :-/

Reply:
yeah, I was able to replicate locally. It's jsoncpp's ABI changes in 1.8.4.

Reply:
conda-forge is currently using 1.8.4 and pins to an exact version (x.x.x). If you cannot build it with 1.8.4 then breaking the pin here for `jsoncpp` is OK. We just need to be sure there are version of the dependecies built with the version you will use here.

Reply:
We can pin jsoncpp 1.8.1 for now. Entwine (and PDAL too) is ditching jsoncpp next release because jsoncpp breaks too much release-to-release.

Reply:
Yep. Got for it!